File: src/C#/StructuralPatterns/Composite/WindowsFormsApplication/Leaves/Button.cs
using System.Drawing;
using WindowsFormsApplication.Leaves.Abstraction;

namespace WindowsFormsApplication.Leaves
{
    public class Button : BaseControlLeaf
    {
        public Button(string name, int width, int height, Color foreColor, Color backColor) 
            : base(name, width, height, foreColor, backColor)
        {

        }
    }
}

File: src/PHP/StructuralPatterns/Composite/WindowsFormsApplication/Leaves/Button.php
<?php

namespace StructuralPatterns\Composite\WindowsFormsApplication\Leaves;

class Button extends BaseControlLeaf
{
    public function __construct($name, $width, $height, $foreColor, $backColor)
    {
        parent::__construct($name, $width, $height, $foreColor, $backColor);
    }
}